Unix/installbuilder/datafiles/Linux.data

```diff
@@ -174,6 +174,34 @@ ConfigureOmiService() {
 %Preinstall_10
 #include OmiService_funcs
 
+# Verify that the proper version of SSL is installed (if we're on ULINUX)
+
+SSL_VERSION=`openssl version | awk '{print $2}'`
```

Comment:
Is awk always in the path on all of our platforms?  One some SunOS system (I think), I recall it being in an awkward path.

Reply:
I thought about that. I asked myself a different question: Can we guarantee that `awk` is even installed on the system?

I used it as is for two reasons:

1. It's already used in `installssllinks`, so there's a dependency on it there, and
2. This was specifically put in `Linux.data`, thus tying it to Linux, where having it installed is quite likely.

In short, we already had a dependency on it for Linux, so I didn't change anything.

Reply:
awk has been around a long time, so I'd imagine it's available everywhere.  I remember that Bruce got into some problems for using some syntax that GNU awk (gawk) supported, but I'm pretty sure all versions of awk support {print $2}.